packages/core/src/physics/PhysicsManager.ts

```diff
@@ -169,6 +167,19 @@ export class PhysicsManager {
     }
   }
 
+  /** The fixed time step in seconds at which physics are performed. */
+  get fixedTimeStep(): number {
+    return this._fixedTimeStep;
+  }
+
+  set fixedTimeStep(value: number) {
+    if (value > 0) {
+      this._fixedTimeStep = value;
+    } else {
+      console.warn("fixedTimeStep must be greater than 0.");
+    }
+  }
```

Comment:
`Math.max(value, MathUtil.zeroTolerance)` is OK

Reply:
resolved